javascript/javascript2/week3/keypresser-Game/main.js

```diff
@@ -0,0 +1,96 @@
+const jsConfetti = new JSConfetti()
+async function confettiDemo(){
+    await jsConfetti.addConfetti()
+    setTimeout(
+    function pageRelode(){
+    window.location.reload();
+    },2000)
+}
+const inputTime = document.getElementById('input-key');
+const btnStart = document.getElementById('btn-start');
+const btnStop = document.getElementById('btn-stop');
+const btnAgain = document.getElementById('btn-again');
+const countdown = document.getElementById('countdown')
+const sKey = document.getElementById('box-1')
+const lKey = document.getElementById('box-2')
+const resultBox = document.getElementById('box-3')
+const winnerResult = document.getElementById("winner");
+const skeyOutput = document.getElementById("skey")
+const lkeyOutput = document.getElementById("lkey")
+let counterS = 0;
+let counterL = 0;
+let countDown = 0;
+function keyPress(e) {
+    if (e.key === 's') {
+        enterTime = inputTime.value;
+        skeyOutput.innerHTML = `Number of Key Pressed ${counterS++}`
```

Comment:
When you add the counter in the string, it will show the counter first, and then will ++ it. So the counting starts at zero (the number of key pressed will be shown zero at first if you pressed the key once) so try to add it first and then show it on the string!

Reply:
yah, you are right, Its showing one count extra in output display. I have updated.